docs/how_to/enabling_access_to_vmpooler.md

```diff
@@ -0,0 +1,18 @@
+# Enabling access to the vmpooler VMs during an acceptance test run
+
+If you are running acceptance tests for Beaker that utilise vmpooler, you will need to run an SSH agent and add the acceptance RSA key prior to running the tests
+
+To load the SSH agent and add the acceptance key, run the following:
+
+~~~bash
+eval `ssh-agent`
+ssh-add ~/.ssh/id_rsa-acceptance
+~~~
+  
+Now run acceptance tests that use the vmpooler:
+
+~~~bash
+bundle exec rake test:subcommands
```

Comment:
I think it might be better to show the Beaker command here, rather than hide it behind "rake".

One question I'm wondering is whether Beaker's `--keyfile` argument is necessary in this case.